packages/fxa-payments-server/src/lib/apiClient.test.ts

```diff
@@ -369,4 +375,96 @@ describe('API requests', () => {
       requestMock.done();
     });
   });
+
+  describe('apiCreateCustomer', () => {
```

Comment:
D'oh. Turns out I left off some apiClient test coverage from the earlier PR for SCA subscription create PR.